### PR TITLE
OpenSSH: New version 9.4.1

### PR DESCRIPTION
**Pull request by eschnett**

### CODE REVIEW

**Review by ararslan — APPROVED** [review]

**Comment by eschnett** [comment]
We are using OpenSSH binaries on Windows, and these were built against OpenSSL 1.1. This PR corrects (i.e. lowers) the compat bound again.

We should also somehow invalidate the current OpenSSH release 9.3.2.

**Comment by mkitti** [comment]
Would the MSYS2 ones work?
https://mirror.msys2.org/msys/x86_64/openssh-9.4p1-1-x86_64.pkg.tar.zst

**Comment by mkitti** [comment]
or maybe we can use their patches:
https://github.com/msys2/MSYS2-packages/tree/master/openssh

**Comment by giordano** [comment]
Uh, if we can build it from source using those patches that'd be great

**Comment by eschnett** [comment]
@mkitti these patches are for OpenSSH @7.3 and don't apply to 9.4.

**Comment by eschnett** [comment]
Also, can we check that these binaries were built against OpenSSL @3?

File: O/OpenSSH/build_tarballs.jl [hunk 1]
```diff
@@ -66,8 +66,11 @@ products = [
 
 # Dependencies that must be installed before this package can be built
 dependencies = [
+    # The Windows binaries (see above) require OpenSSL @1.1;
```
[the remark this answers is not in the document]
Comment (eschnett) [reply O/OpenSSH/build_tarballs.jl]: Thanks.

**Comment by mkitti** [comment]
I'm realizing that the openssh package I pointed to are MSYS2 packages rather MINGW packages so they may not be as helpful asI would have thought.

**Comment by eschnett** [comment]
I don't know what this means. Can we still use them? We only want executables; would they still run?

**Comment by imciner2** [comment]
Closing due to inactivity. Please submit a new PR on top of the current master branch if you still want to include this recipe.

**Pull request closed without merging**